doc/command-line.md

```diff
@@ -0,0 +1,138 @@
+
+# Using sourmash from the command line
+
+
+From the command line, sourmash can be used to compute [MinHash
+sketches][0] from DNA
+sequences, compare them to each other, and plot the results.  This
+allows you to estimate sequence similarity quickly and accurately.
+
+Please see the [mash software][1] http://mash.readthedocs.io/en/latest/__
+ and the [mash paper (Ondov et al., 2016)][2]
+for background
+information on how and why MinHash sketches work.
+
+______
+
+sourmash uses a subcommand syntax, so all commands start with
+`sourmash` followed by a subcommand specifying the action to be
+taken.
+
+Contents
+
+* [An example](## An example)
+* [The `sourmash` command and its subcommands](## The sourmash command and its subcommands)
+  * [`sourmash compute`](### `sourmash compute`)
+
+  * [`sourmash compare`](### `sourmash compare`)
+
+  * [`sourmash plot`](### `sourmash plot`)
+
+## An example
+
+
+Grab three bacterial genomes from NCBI:
+```
+   curl -L -O ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Escherichia_coli/reference/GCF_000005845.2_ASM584v2/GCF_000005845.2_ASM584v2_genomic.fna.gz
+   curl -L -O ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Salmonella_enterica/reference/GCF_000006945.1_ASM694v1/GCF_000006945.1_ASM694v1_genomic.fna.gz
+   curl -L -O ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/bacteria/Sphingobacteriaceae_bacterium_DW12/latest_assembly_versions/GCF_000783305.1_ASM78330v1/GCF_000783305.1_ASM78330v1_genomic.fna.gz
+```
+Compute signatures for each:
+```
+   sourmash compute *.fna.gz
+```
+This will produce three `.sig` files containing MinHash signatures at k=31.
+
+Next, compare all the signatures to each other:
+```
+   sourmash compare *.sig -o cmp
+```
+Finally, plot a dendrogram:
+```
+   sourmash plot cmp
+```
+This will output two files, `cmp.dendro.png` and `cmp.matrix.png`,
+containing a clustering & dendrogram of the sequences, as well as a
+similarity matrix and heatmap.
+
+## The `sourmash` command and its subcommands
+
+
+To get a list of subcommands, run `sourmash` without any arguments.
+
+There are three main subcommands: `compute`, `compare`, and `plot`.
+
+### `sourmash compute`
+
+
+The `compute` subcommand computes and saves MinHash sketches for
+each sequence in one or more sequence files.  It takes as input FASTA
+or FASTQ files, and these files can be uncompressed or compressed with
+gzip or bzip2.  The output will be one or more JSON signature files
+that can be used with `sourmash compare`.
+
+Usage:
```

Comment:
Also a copy/paste mistake. Instead of typing this out fully, this should be auto-generated with the `.. Usage::` directive. That way changes to the code are automatically reflected in the documentation.

Reply:
My bad, I was mistaken. This is not a directive.